### PR TITLE
Feat/skfp 934/cavatica bulk import

### PR DESCRIPTION
**Pull request by lflangis**
#[BUG | FEATURE | DOC] [TITLE]

- closes #[895](https://d3b.atlassian.net/browse/SKFP-895)
- closes #[934](https://d3b.atlassian.net/browse/SKFP-934)

## Description
I added more metadata when CavaticaBulkImport is triggered.

## Screenshot 
image
image

### CODE REVIEW

File: src/components/Cavatica/AnalyzeButton/index.tsx [hunk 1]
```diff
@@ -66,12 +67,14 @@ const CavaticaAnalyzeButton: React.FC<OwnProps> = ({
   }, [cavatica.authentification.status, cavatica.bulkImportData.status, onBeginAnalyse]);
 
   return (
+    //TODO add spinner when loading modal
```
Comment (evans-g-crsj) [inline src/components/Cavatica/AnalyzeButton/index.tsx:70]: You can delete this comment

File: src/graphql/utils/helpers.test.ts [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  test('should handle edge case', () => {
+    expect(
+      toNodes({
+        hits: {
+          edges: [],
+        },
+      }),
+    ).toEqual([]);
+  });
+  test('should return first node with truthy target prop when exists', () => {
```
Comment (evans-g-crsj) [inline src/graphql/utils/helpers.test.ts:13]: My bad, could you change it to:
`should return nodes content`

File: src/components/utils/NotificationContextHolder/index.tsx [hunk 3]
```diff
@@ -15,6 +17,9 @@ const NotificationContextHolder = () => {
     if (notification) {
       antNotification.open({
         ...notification,
+        description: React.createElement('div', {
```
Comment (evans-g-crsj) [inline src/components/utils/NotificationContextHolder/index.tsx:20]: could you add a comment that says that: `notification.description` is local code (as opposed to an external server)?

File: src/store/passport/thunks.ts [hunk 4]
```diff
@@ -253,6 +264,11 @@ export const beginCavaticaAnalyse = createAsyncThunk<
     variables: {
       sqon,
       first: CAVATICA_FILE_BATCH_SIZE,
+      //TODO s
```
Comment (evans-g-crsj) [inline src/store/passport/thunks.ts:267]: remove those comments, pls.

**Review by evans-g-crsj — CHANGES_REQUESTED** [review]
Just need to clean up some outdated comments pls

**Comment by evans-g-crsj** [comment]
I added more metadata when CavaticaBulkImport is triggered.

**Review by evans-g-crsj — APPROVED** [review]
